scripts/live_test/CLITest.yml

```diff
@@ -176,16 +186,35 @@ jobs:
         pip install pytest-json-report
         pip install pytest-html
         pip install pytest-rerunfailures
+
         azdev setup -c azure-cli
+
         # az login --service-principal --username $(APP_ID) --password $(PASSWORD) --tenant $(TENANT_ID)
         az login -u azureclitest@azuresdkteam.onmicrosoft.com -p $(AZURECLITESTPASSWORD)
         az account set -s 0b1f6471-1bf0-4dda-aec3-cb9272f09590
-        # Sequential
-        azdev test $(Target) $(USER_LIVE) --mark serial --xml-path test_results.sequential.xml --no-exitfirst -a "-n 1 --json-report --json-report-summary --json-report-file=$(Target).report.sequential.json --html=$(Target).report.sequential.html --self-contained-html --reruns 3 -s"
-        # Parallel
-        azdev test $(Target) $(USER_LIVE) --mark "not serial" --xml-path test_results.parallel.xml --no-exitfirst -a "-n $(USER_PARALLELISM) --json-report --json-report-summary --json-report-file=$(Target).report.parallel.json --html=$(Target).report.parallel.html --self-contained-html --reruns 3 -s"
+
+        # Commit changes
+        if [[ "$(USER_USERNAME)" != "" || "$(USER_TOKEN)" != "" ]]; then
+          azdev test $(Target) --no-exitfirst -a "-n $(USER_PARALLELISM)"
+          azdev test $(Target) --live --lf --xml-path test_results.parallel.xml --no-exitfirst -a "-n $(USER_PARALLELISM) --json-report --json-report-summary --json-report-file=$(Target).report.parallel.json --html=$(Target).report.parallel.html --self-contained-html --reruns 3 -s"
```

Comment:
Just to point out that some ADO agent only has 2 cores and the current default USER_PARALLELISM=8 will not be run as expected.

Reply:
2 cores does not mean it can only run 2 processes.
https://en.wikipedia.org/wiki/Context_switch

Reply:
Logs in pipeline
```
RUNNING TESTS LIVE
The tests are set to run against current profile "latest"
============================= test session starts ==============================
platform linux -- Python 3.8.5, pytest-6.0.2, py-1.9.0, pluggy-0.13.1 -- /home/vsts/work/1/env/bin/python
[gw0] linux Python 3.8.5 cwd: /home/vsts/work/1

[gw1] linux Python 3.8.5 cwd: /home/vsts/work/1

[gw2] linux Python 3.8.5 cwd: /home/vsts/work/1

[gw3] linux Python 3.8.5 cwd: /home/vsts/work/1

[gw4] linux Python 3.8.5 cwd: /home/vsts/work/1

[gw5] linux Python 3.8.5 cwd: /home/vsts/work/1

[gw6] linux Python 3.8.5 cwd: /home/vsts/work/1

[gw7] linux Python 3.8.5 cwd: /home/vsts/work/1

[gw0] Python 3.8.5 (default, Jul 21 2020, 12:22:34)  -- [GCC 7.5.0]

[gw1] Python 3.8.5 (default, Jul 21 2020, 12:22:34)  -- [GCC 7.5.0]

[gw2] Python 3.8.5 (default, Jul 21 2020, 12:22:34)  -- [GCC 7.5.0]

[gw3] Python 3.8.5 (default, Jul 21 2020, 12:22:34)  -- [GCC 7.5.0]

[gw4] Python 3.8.5 (default, Jul 21 2020, 12:22:34)  -- [GCC 7.5.0]

[gw5] Python 3.8.5 (default, Jul 21 2020, 12:22:34)  -- [GCC 7.5.0]

[gw6] Python 3.8.5 (default, Jul 21 2020, 12:22:34)  -- [GCC 7.5.0]

[gw7] Python 3.8.5 (default, Jul 21 2020, 12:22:34)  -- [GCC 7.5.0]
gw0 [216] / gw1 [216] / gw2 [216] / gw3 [216] / gw4 [216] / gw5 [216] / gw6 [216] / gw7 [216]
```